static/js/biomedical/protein/protein_protein_interaction_graph.tsx

```diff
@@ -104,19 +119,67 @@ export class ProteinProteinInteractionGraph extends React.Component<
     return (
       <>
         <div id={CHART_ID}></div>
-        <FormGroup>
-          <Label for={DEPTH_INPUT_ID}>Depth</Label>
-          <Input
-            id={DEPTH_INPUT_ID}
-            type="number"
-            min={MIN_DEPTH}
-            max={MAX_DEPTH}
-            onChange={(e) => {
-              this.setState({ depth: Number(e.target.value) });
-            }}
-            value={this.state.depth}
-          />
-        </FormGroup>
+        <Row>
+          <Col md={2}>
+            <FormGroup>
+              <Label for={DEPTH_INPUT_ID}>Depth</Label>
+              <Input
+                id={DEPTH_INPUT_ID}
+                className="ppi-input"
+                type="number"
+                min={MIN_DEPTH}
+                max={MAX_DEPTH}
+                onChange={(e) => {
+                  this.setState({ depth: Number(e.target.value) });
+                }}
+                value={this.state.depth}
+              />
+            </FormGroup>
+          </Col>
+          <Col md={2}>
+            <FormGroup>
+              <Label for={NUM_INTERACTORS_INPUT_ID}>Interactors</Label>
+              <Input
+                id={NUM_INTERACTORS_INPUT_ID}
+                className="ppi-input"
+                type="number"
+                min={MIN_NUM_INTERACTORS}
+                max={MAX_NUM_INTERACTORS}
+                onChange={(e) => {
+                  this.setState({ numInteractions: Number(e.target.value) });
+                }}
+                value={this.state.numInteractions}
+              />
+            </FormGroup>
+          </Col>
+          <Col md={3}>
+            <FormGroup>
+              <Label for={SCORE_THRESHOLD_INPUT_ID}>Confidence Threshold</Label>
+              <Input
+                id={SCORE_THRESHOLD_INPUT_ID}
+                className="ppi-input"
+                type="number"
+                min={MIN_SCORE_THRESHOLD}
+                max={MAX_SCORE_THRESHOLD}
+                step={0.1}
+                onChange={(e) => {
+                  this.setState({ scoreThreshold: Number(e.target.value) });
+                }}
+                value={this.state.scoreThreshold}
+              />
+            </FormGroup>
+          </Col>
+          <Col className="form-group align-self-end" md={2}>
```

Comment:
I don't think these class names are necessary since your'e not doing anything with them

Reply:
Oh I forgot to link a reference but I think these might be handled by reactstrap - before adding these the button was vertically aligning with the labels instead of the text fields as in [https://stackoverflow.com/questions/40878566/align-button-with-input-forms-with-labels](here).  What do you think?

Reply:
ok, fine to keep then.

Reply:
thanks a lot! will do